R/type-factor.R

```diff
@@ -223,9 +223,9 @@ hash_label <- function(x, length = 5) {
   if (length(x) == 0) {
     ""
   } else {
-    # Can't use hash() currently because it hashes the string pointers
+    # Can't use obj_hash() because it hashes the string pointers
     # for performance, so the values in the test change each time
-    substr(digest::digest(x), 1, length)
+    substr(rlang::hash(x), 1, length)
```

Comment:
Probably worth doing a revdep run just to make sure this doesn't break any existing tests.

Reply:
I revdep run shows no breakage!